modules/hostap/Kconfig

```diff
@@ -16,11 +16,16 @@ menuconfig WPA_SUPP
 	select NET_L2_WIFI_MGMT
 	select WIFI_NM
 	select EXPERIMENTAL if !SOC_SERIES_NRF53X && !SOC_SERIES_NRF91X
+	select COMMON_LIBC_MALLOC
 	help
 	  WPA supplicant implements 802.1X related functions.
 
 if WPA_SUPP
 
+config COMMON_LIBC_MALLOC_ARENA_SIZE
+	# 30K is mandatory, but might need more for long duration use cases
```

Comment:
What is the meaning of "long duration" use cases , here @krish2718 ?

Reply:
12hours stress tests that we have nightly and we are seeing occasional OOM messages randomly (4/6/9hrs), but running with 40K is helping tests pass. But we still don't know the root cause of the heap crunch.

Reply:
Thanks. @krish2718 it would be nice if the help text here explains these points
- "30 kB is mandatory" (mandatory for what? Should we say this is minimum requirement/baseline?)
- "A higher amount of heap may be required for long duration stress-tests" Although I've go doubts is this is the right thing to do: I can definitely think of customer scenarios that would require long uptime for the Wi-Fi stack :)

Wouldn't it be safer to determine the right size of the MALLOC arena that is OK for all mainline cases?

Reply:
Great. Just improve a bit the help text, for the Kconfig, if possible